test/metabase/query_processor_test/date_time_zone_functions_test.clj

```diff
@@ -943,12 +954,11 @@
   (mt/test-drivers (->> (disj (mt/normal-drivers-with-feature :datetime-diff) :snowflake)
                         (filter mt/supports-time-type?))
     (testing "Cannot datetime-diff against time column"
-      (mt/dataset attempted-murders
+      (mt/dataset test-data-with-time
```

Comment:
Change 6: We have to use a different dataset that has a time column but no timestamptz column, because there are some drivers that only support time but not timestamptz.